src/components/link/link.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Link, { InternalLink } from '.';
+
+<Meta title="Link" parameters={{ info: { disable: true }}} />
+
+# Link
+Navigates the user to another location.
+
+Avoid using links for commands (performing an action like saving a form) - use them for navigating the user to a new location. 
+If you’re navigating the user away from your site, consider using an icon after the link, and open the link in a new window, so the user doesn’t lose their place.
+To make the meaning of a link clearer, you can add an icon before it. Just name one of the Carbon icons.
+Make your link names meaningful - for example, instead of ‘Click here’, try ‘Download Invoice 001’.
+WCAG guidelines recommend that Color is not used as the only visual means of conveying information, indicating an action. Carbon applies bold weight, but you could also use text decoration.
+
+## Quick Start
+Import `Link` into the project.
+
+```jsx
+import Link from "carbon-react/lib/components/link";
+
+const MyComponent = () => (
+  <Link>
+    This is a link
+  </Link>
+);
+```
+
+### Default
+<Preview>
+  <Story name="Default">
+    <Link>
+      This is a link
+    </Link>
+  </Story>
+</Preview>
+
+### Disabled
+Setting the `disabled` prop to true disables the link.
+
+<Preview>
+  <Story name="with Disabled">
+    <Link disabled>
+      This is a link
+    </Link>
+  </Story>
+</Preview>
+
+### href
+The `href` prop provides the ability for the Link to take the user to a different webpage or another part of the DOM.
+
+<Preview>
+  <Story name="with href">
+    <Link href='https://carbon.sage.com'>
+      This is a link
+    </Link>
+  </Story>
+</Preview>
+
+### Icon
+The `icon` prop allows you to render an icon.
```

Comment:
we should ignore this `chromatic`, we have below `Icon Align` story also with Icons